stdlib/http/http.go

```diff
@@ -0,0 +1,235 @@
+package http
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/iocounter"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// maxResponseBody is the maximum response body we will read before just discarding
+// the rest. This allows sockets to be reused.
+const maxResponseBody = 512 * 1024 // 512 KB
+
+var httpGetStatusCodeOnly = values.NewFunction(
+	"get",
+	semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"url":     semantic.String,
+			"headers": semantic.Tvar(1),
+			"data":    semantic.Bytes,
```

Comment:
As per your comment below do we need the `data` parameter to this function? Does your use case need it? If not then I say lets remove it until its clear we need it.

Reply:
Fair enough. I was torn since the Go client accepted and used data that perhaps I should include it, but that isn't overly compelling.